src/Codec/Xlsx/Types/DataValidation.hs

```diff
@@ -183,20 +216,39 @@ readValidationType op ty cur = do
     opExp <- readOpExpression2 op cur
     readValidationTypeOpExp ty opExp
 
-readListFormulas :: Formula -> Maybe [Text]
-readListFormulas (Formula f) = readQuotedList f
+-- | Attempt to obtain a range expression from the list of ValidationTypeList
+getCellRangeValidator :: ListOrRangeExpression -> Maybe Range
```

Comment:
what do you think about using `maybe` instead of `get`? And the same for `getPlainListValidator` (I'd also rename it to `maybePlainValidationList` which is practically its return type :) )

Reply:
Also `ValidationTypeList` in the comment looks to be a bit out of scope as we don't deal with `ValidationType` here...

Reply:
I have barely migrated these from my previous attempt conserving `| ValidationTypeList [Text]` 😅 

Perhaps they'd be more useful taking `ValidationType` to match `ValidationTypeList` ?

It would also fit the names you've suggested :)  

``` haskell
-- | Attempt to obtain a plain list from a @ValidationTypeList@
maybePlainValidationList :: ValidationType -> Maybe ValidationList
```
``` haskell
-- | Attempt to obtain a cell or range reference from a @ValidationTypeList@
maybeRangeValidationList :: ValidationType -> Maybe Range
```

or removing them altogether

Reply:
either way is fine with me